examples/windows_nodepool_with_private_cluster/gke/main.tf

```diff
@@ -0,0 +1,95 @@
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "gke_private_cluster" {
+  source                     = "../modules/google_kubernetes_engine/gke_private_cluster/"
+  project_id                 = var.project
+  name                       = "test-gke-cluster"
```

Comment:
nit: we should give it a descriptive name, perhaps similar to our existing example naming conventions
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/26ea28db52ce5797da043a1c56dfb4575f49dac8/examples/simple_regional_private/main.tf#L35

Reply:
done added proper naming convention